### ISSUE TITLE
Dependencies / Behaviour on alpine

### ISSUE THREAD

**Issue by sharedstuff** [dfinke/ImportExcel#1469]
Hi Doug,

got the following problem:

WARNING: ImportExcel Module Cannot Autosize. Please run the following command to install dependencies:
apt-get -y update && apt-get install -y --no-install-recommends libgdiplus libc6-dev

Environment:
alpine powershell container

Behaviour/Problems:
Warning spawns even when AutoSize is even explicitely set to $false and WarningAction is set to SilentlyContinue
Installing the alpine equivalents to libgdiplus/libc6-dev doesnt seem to fix it

How to approach/troubleshoot?

Regards, Machete83 in disguise :)

**Comment by dfinke** [comment]
Yeah. Please search the issues. That issue has been around for a while. I don’t recall if there was a workaround

**Comment by sharedstuff** [comment]
https://github.com/dfinke/ImportExcel/blob/ab4d03c984fc7a8b129c8fe03af02b1f29f743e5/ImportExcel.psm1#L32


if (($IsLinux -or $IsMacOS) -or $env:NoAutoSize) {
->
if (-not $env:NoAutoSize -and ($IsLinux -or $IsMacOS)) {

and should be "not-working as intended at least"
you are "trying" an autosize even if env is set to dont do it ... 

Still dont get why it simply works out
libgdiplus is around
libc6-dev = dotnet6 sdk or am i wrong?
